internal/auth/authenticator_test.go

```diff
@@ -71,7 +71,7 @@ var testEncodedCookieSecret = "x7xzsM1Ky4vGQPwqy6uTztfr3jtm/pIdRbJXgE0q8kU="
 var testAuthCodeSecret = "qICChm3wdjbjcWymm7PefwtPP6/PZv+udkFEubTeE38="
 
 func testOpts(proxyClientID, proxyClientSecret string) *Options {
-	opts := NewOptions()
+	opts, _ := NewOptions()
```

Comment:
My preference would be to check the err here if possible. In the case of these tests, let's pass a `t *testing.T` and fatal if these options aren't instantiated correctly.